src/views/XBlockStudioView/components/StudioSchedule/StepInfo.test.jsx

```diff
@@ -1,17 +1,55 @@
-import { shallow } from '@edx/react-unit-test-utils';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { IntlProvider } from '@edx/frontend-platform/i18n';
 
 import StepInfo from './StepInfo';
 
-jest.mock('./FormatDateTime', () => 'FormatDateTime');
+jest.unmock('@openedx/paragon');
+jest.unmock('react');
+jest.unmock('@edx/frontend-platform/i18n');
+
+jest.mock('./FormatDateTime', () => {
+  // eslint-disable-next-line react/prop-types
+  const FormatDateTime = ({ date }) => <span>{date || 'Not set'}</span>;
+  return FormatDateTime;
+});
 
 describe('<StepInfo />', () => {
-  it('should render', () => {
-    const wrapper = shallow(<StepInfo stepName="test" />);
-    expect(wrapper.snapshot).toMatchSnapshot();
+  const renderWithIntl = (component) => render(<IntlProvider locale="en">{component}</IntlProvider>);
+
+  it('renders step info without dates', () => {
+    renderWithIntl(<StepInfo stepName="test" />);
+    expect(screen.queryByText('test start:')).not.toBeInTheDocument();
+    expect(screen.queryByText('test due:')).not.toBeInTheDocument();
+  });
+
+  it('renders step info with start and end dates', () => {
+    renderWithIntl(
+      <StepInfo
+        stepName="test"
+        startDatetime="2020-01-01T00:00:00Z"
+        endDatetime="2020-01-02T00:00:00Z"
+      />,
+    );
+    expect(screen.getByText('test start:')).toBeInTheDocument();
+    expect(screen.getByText('test due:')).toBeInTheDocument();
+    expect(screen.getByText('2020-01-01T00:00:00Z')).toBeInTheDocument();
+    expect(screen.getByText('2020-01-02T00:00:00Z')).toBeInTheDocument();
+  });
+
+  it('renders only start date when end date is not provided', () => {
+    renderWithIntl(
+      <StepInfo stepName="test" startDatetime="2020-01-01T00:00:00Z" />,
+    );
+    expect(screen.getByText('test start:')).toBeInTheDocument();
+    expect(screen.queryByText('test due:')).not.toBeInTheDocument();
   });
```

Comment:
The "`renders step info with start and end dates`" test has `expect`s for the actual rendering of the start/end datetimes. This is only checking the `test start:` text is rendered and the `test due:` test is not rendered. Should this also check to make sure the test start datetime is rendered?